azure-pipelines.yml

```diff
@@ -557,7 +557,7 @@ jobs:
       set -e
       . util/build_consts.sh
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-      ci/scripts/run-fpga-cw310-tests.sh cw310_rom,jtag || { res=$?; echo "To reproduce failures locally, follow the instructions at https://docs.opentitan.org/doc/getting_started/setup_fpga/#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
+      ci/scripts/run-fpga-cw310-tests.sh jtag || { res=$?; echo "To reproduce failures locally, follow the instructions at https://docs.opentitan.org/doc/getting_started/setup_fpga/#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```

Comment:
If we had any non-CW310 "jtag" tests, they would sneak in here. I'm suppressing my inner perfectionist, but I'd like to be able to specify `cw310_rom AND jtag`.

Reply:
I'd have expected cw310_rom,jtag to work

Reply:
Yeah, it's subtle! Here's the tricky bit from the Bazel [docs](https://bazel.build/reference/command-line-reference#flag--test_tag_filters).

> Only those test targets will be found that contain at least one included tag and do not contain any excluded tags. This option affects --build_tests_only behavior and the test command.

Reply:
Unsurprisingly, this came up before. https://github.com/bazelbuild/bazel/issues/3705 seems to describe exactly the same pain point. They wrote a few PRs that ultimately were not merged.